### ISSUE TITLE
2021-02

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#17]

**Comment by qiaojy19** [comment]
# Feb - 2021

## 02.22 - 02.26

| Title                                                                                                                                       |
| ------------------------------------------------------------------------------------------------------------------------------------------- |
| [Toward Understanding the Influence of Individual Clients in Federated Learning](https://arxiv.org/abs/2012.10936)                          |
| [Federated f-Differential Privacy](https://arxiv.org/abs/2102.11158)                                                                        |
| [Federated Multi-armed Bandits with Personalization](https://arxiv.org/abs/2102.13101)                                                      |
| [Clustering Algorithm to Detect Adversaries in Federated Learning](https://arxiv.org/abs/2102.10799)                                        |
| [Efficient Client Contribution Evaluation for Horizontal Federated Learning](https://arxiv.org/abs/2102.13314)                              |
| [Achieving Linear Speedup with Partial Worker Participation in Non-IID Federated Learning](https://arxiv.org/abs/2101.11203)                |
| [Privacy Preserving Distributed Machine Learning with Federated Learning](https://arxiv.org/abs/2004.12108)                                 |
| [Towards Flexible Device Participation in Federated Learning](https://arxiv.org/abs/2006.06954)                                             |
| [Free-rider Attacks on Model Aggregation in Federated Learning](https://arxiv.org/abs/2006.11901)                                           |
| [Distributed Saddle-Point Problems: Lower Bounds, Optimal Algorithms and Federated GANs](https://arxiv.org/abs/2010.13112)                  |
| [Ditto: Fair and Robust Federated Learning Through Personalization](https://arxiv.org/abs/2012.04221)                                       |
| [Multiple Kernel-Based Online Federated Learning](https://arxiv.org/abs/2102.10861)                                                         |
| [Sustainable Federated Learning](https://arxiv.org/abs/2102.11274)                                                                          |
| [Federated Learning for Physical Layer Design](https://arxiv.org/abs/2102.11777)                                                            |
| [QuPeL: Quantized Personalization with Applications to Federated Learning](https://arxiv.org/abs/2102.11786)                                |
| [Lossless Compression of Efficient Private Local Randomizers](https://arxiv.org/abs/2102.12099)                                             |
| [Estimation of Continuous Blood Pressure from PPG via a Federated Learning Approach](https://arxiv.org/abs/2102.12245)                      |
| [Wirelessly Powered Federated Edge Learning: Optimal Tradeoffs Between Convergence and Power Transfer](https://arxiv.org/abs/2102.12357)    |
| [Distributed Online Learning with Multiple Kernels](https://arxiv.org/abs/2102.12733)                                                       |
| [Distributed Online Learning with Multiple Kernels](https://arxiv.org/abs/2102.12733)                                                       |
| [Blockchained Federated Learning for Threat Defense](https://arxiv.org/abs/2102.12746)                                                      |
| [Emerging Trends in Federated Learning: From Model Fusion to Federated X Learning](https://arxiv.org/abs/2102.12920)                        |
| [The Magic of Superposition: A Survey on the Simultaneous Transmission Based Wireless Systems](https://arxiv.org/abs/2102.13144)            |
| [Cybersecurity Threats in Connected and Automated Vehicles based](https://arxiv.org/abs/2102.13256)                                         |
| [FjORD: Fair and Accurate Federated Learning under heterogeneous targets with Ordered Dropout](https://arxiv.org/abs/2102.13451)            |
| [Federated Edge Learning with Misaligned Over-The-Air Computation](https://arxiv.org/abs/2102.13604)                                        |
| [Practical and Private (Deep) Learning without Sampling or Shuffling](https://arxiv.org/abs/2103.00039)                                     |
| [Scalable federated machine learning with FEDn](https://arxiv.org/abs/2103.00148)                                                           |
| [Constrained Differentially Private Federated Learning for Low-bandwidth Devices](https://arxiv.org/abs/2103.00342)                         |
| [Integrating Over-the-Air Federated Learning and Non-Orthogonal Multiple Access: What Role can RIS Play?](https://arxiv.org/abs/2103.00435) |
| [Federated Learning without Revealing the Decision Boundaries](https://arxiv.org/abs/2103.00695)                                            |
| [Heterogeneity for the Win: One-Shot Federated Clustering](https://arxiv.org/abs/2103.00697)                                                |
| [Privacy-Preserving Distributed SVD via Federated Power](https://arxiv.org/abs/2103.00704)                                                  |
| [Towards Personalized Federated Learning](https://arxiv.org/abs/2103.00710)                                                                 |
| [Distributionally Robust Federated Averaging](https://arxiv.org/abs/2102.12660)                                                             |
## 02.15 - 02.19

| Title                                                                                                                                                      |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Provably Secure Federated Learning against Malicious Clients](https://arxiv.org/abs/2102.01854)                                                           |
| [Estimation of Microphone Clusters in Acoustic Sensor Networks using Unsupervised Federated Learning](https://arxiv.org/abs/2102.03109)                    |
| [Federated Dropout Learning for Hybrid Beamforming With Spatial Path Index Modulation In Multi-User mmWave-MIMO Systems](https://arxiv.org/abs/2102.07450) |
| [FedBN: Federated Learning on Non-IID Features via Local Batch Normalization](https://arxiv.org/abs/2102.07623)                                            |
| [Scaling Neuroscience Research using Federated Learning](https://arxiv.org/abs/2102.08440)                                                                 |
| [Making a Case for Federated Learning in the Internet of Vehicles and Intelligent Transportation Systems](https://arxiv.org/abs/2102.10142)                |
| [Hybrid Federated Learning: Algorithms and Implementation](https://arxiv.org/abs/2012.12420)                                                               |
| [A Federated Data-Driven Evolutionary Algorithm](https://arxiv.org/abs/2102.08288)                                                                         |
| [A first look into the carbon footprint of federated learning](https://arxiv.org/abs/2102.07627)                                                           |
| [Byzantine-Resilient Secure Federated Learning](https://arxiv.org/abs/2007.11115)                                                                          |
| [CFLMEC: Cooperative Federated Learning for Mobile Edge Computing](https://arxiv.org/abs/2102.10591)                                                       |
| [Characterizing Impacts of Heterogeneity in Federated Learning upon Large-Scale Smartphone Data](https://arxiv.org/abs/2006.06983)                         |
| [CSIT-Free Federated Edge Learning via Reconfigurable Intelligent Surface](https://arxiv.org/abs/2102.10749)                                               |
| [Data-Aware Device Scheduling for Federated Edge Learning](https://arxiv.org/abs/2102.09491)                                                               |
| [DESED-FL and URBAN-FL: Federated Learning Datasets for Sound Event Detection](https://arxiv.org/abs/2102.08833)                                           |
| [Distributed Saddle-Point Problems: Lower Bounds, Optimal Algorithms and Federated GANs](https://arxiv.org/abs/2010.13112)                                 |
| [Edge Federated Learning Via Unit-Modulus Over-The-Air Computation](https://arxiv.org/abs/2101.12051)                                                      |
| [Efficient Reinforcement Learning in Resource Allocation Problems Through Permutation Invariant Multi-task Learning](https://arxiv.org/abs/2102.09361)     |
| [Faster Non-Convex Federated Learning via Global and Local Momentum](https://arxiv.org/abs/2012.04061)                                                     |
| [Federated Composite Optimization](https://arxiv.org/abs/2011.08474)                                                                                       |
| [Federated Evaluation and Tuning for On-Device Personalization: System Design & Applications](https://arxiv.org/abs/2102.08503)                            |
| [Federated Learning over Wireless Networks: A Band-limited Coordinated Descent Approach](https://arxiv.org/abs/2102.07972)                                 |
| [Federated Reconstruction: Partially Local Federated Learning](https://arxiv.org/abs/2102.03448)                                                           |
| [Federated Unlearning](https://arxiv.org/abs/2012.13891)                                                                                                   |
| [HeteroFL: Computation and Communication Efficient Federated Learning for Heterogeneous Clients](https://arxiv.org/abs/2010.01264)                         |
| [Hybrid Federated and Centralized Learning](https://arxiv.org/abs/2011.06892)                                                                              |
| [Label Leakage and Protection in Two-party Split Learning](https://arxiv.org/abs/2102.08504)                                                               |
| [MARINA: Faster Non-Convex Distributed Learning with Compression](https://arxiv.org/abs/2102.07845)                                                        |
| [Mobility-Aware Routing and Caching: A Federated Learning Assisted Approach](https://arxiv.org/abs/2102.10743)                                             |
| [On the Impact of Device and Behavioral Heterogeneity in Federated Learning](https://arxiv.org/abs/2102.07500)                                             |
| [Personalized Federated Learning: A Unified Framework and Universal Optimization Techniques](https://arxiv.org/abs/2102.09743)                             |
| [Policy-Based Federated Learning](https://arxiv.org/abs/2003.06612)                                                                                        |
| [ppAUC: Privacy Preserving Area Under the Curve with Secure 3-Party Computation](https://arxiv.org/abs/2102.08788)                                         |
| [Privacy-Preserving Wireless Federated Learning Exploiting Inherent Hardware Impairments](https://arxiv.org/abs/2102.10639)                                |
| [Self-supervised Cross-silo Federated Neural Architecture Search](https://arxiv.org/abs/2101.11896)                                                        |
| [Turbo-Aggregate: Breaking the Quadratic Aggregation Barrier in Secure](https://arxiv.org/abs/2002.04156)                                                  |
| [Voting-based Approaches For Differentially Private Federated Learning](https://arxiv.org/abs/2010.04851)                                                  |
| [WAFFLE: Watermarking in Federated Learning](https://arxiv.org/abs/2008.07298)                                                                             |
| [When Crowdsensing Meets Federated Learning: Privacy-Preserving Mobile Crowdsensing System](https://arxiv.org/abs/2102.10109)                              |
## 02.08 - 02.12
| Title                                                                                                                                            |
| ------------------------------------------------------------------------------------------------------------------------------------------------ |
| [Adaptive Quantization of Model Updates for Communication-Efficient Federated Learning](https://arxiv.org/abs/2102.04487)                        |
| [Decentralizing Feature Extraction with Quantum Convolutional Neural Network for Automatic Speech Recognition](https://arxiv.org/abs/2010.13309) |
| [Federated Acoustic Modeling For Automatic Speech Recognition](https://arxiv.org/abs/2102.04429)                                                 |
| [Federated Learning with Local Differential Privacy: Trade-offs between Privacy, Utility, and Communication](https://arxiv.org/abs/2102.04737)   |
| [Private Wireless Federated Learning with Anonymous Over-the-Air Computation](https://arxiv.org/abs/2011.08579)                                  |
| [A Metamodel and Framework for Artificial General Intelligence From Theory to Practice](https://arxiv.org/abs/2102.06112)                        |
| [Achieving Linear Convergence in Federated Learning under Objective and Systems Heterogeneity](https://arxiv.org/abs/2102.07053)                 |
| [CaPC Learning: Confidential and Private Collaborative Learning](https://arxiv.org/abs/2102.05188)                                               |
| [Distributed Networked Learning with Correlated Data](https://arxiv.org/abs/1910.12783)                                                          |
| [Distributed Spectrum and Power Allocation for D2D-U Networks: A Scheme based on NN and Federated Learning](https://arxiv.org/abs/2102.04359)    |
| [Edge Bias in Federated Learning and its Solution by Buffered Knowledge Distillation](https://arxiv.org/abs/2010.10338)                          |
| [Efficient Algorithms for Federated Saddle Point Optimization](https://arxiv.org/abs/2102.06333)                                                 |
| [Enhancing WiFi Multiple Access Performance with Federated Deep Reinforcement Learning](https://arxiv.org/abs/2102.07019)                        |
| [Exploiting Shared Representations for Personalized Federated Learning](https://arxiv.org/abs/2102.07078)                                        |
| [FedCM: A Real-time Contribution Measurement Method for Participants in Federated Learning](https://arxiv.org/abs/2009.03510)                    |
| [Federated Deep AUC Maximization for Heterogeneous Data with a Constant Communication Complexity](https://arxiv.org/abs/2102.04635)              |
| [Federated Learning of a Mixture of Global and Local Models](https://arxiv.org/abs/2002.05516)                                                   |
| [FedGNN: Federated Graph Neural Network for Privacy-Preserving Recommendation](https://arxiv.org/abs/2102.04925)                                 |
| [FedU: A Unified Framework for Federated Multi-Task Learning with Laplacian Regularization](https://arxiv.org/abs/2102.07148)                    |
| [FLOP: Federated Learning on Medical Datasets using Partial Networks](https://arxiv.org/abs/2102.05218)                                          |
| [Meta Federated Learning](https://arxiv.org/abs/2102.05561)                                                                                      |
| [Privacy Amplification by Decentralization](https://arxiv.org/abs/2012.05326)                                                                    |
| [Privacy-Preserving Self-Taught Federated Learning for Heterogeneous Data](https://arxiv.org/abs/2102.05883)                                     |
| [Proximal and Federated Random Reshuffling](https://arxiv.org/abs/2102.06704)                                                                    |
| [Robust Federated Learning with Attack-Adaptive Aggregation](https://arxiv.org/abs/2102.05257)                                                   |
| [Security and Privacy for Artificial Intelligence: Opportunities and Challenges](https://arxiv.org/abs/2102.04661)                               |
| [Specialized federated learning using a mixture of experts](https://arxiv.org/abs/2010.02056)                                                    |
| [Stragglers Are Not Disaster: A Hybrid Federated Learning Algorithm with Delayed Gradients](https://arxiv.org/abs/2102.06329)                    |
| [Task-oriented Communication System Design in Cyber-Physical Systems: A Survey on Theory and Applications](https://arxiv.org/abs/2102.07166)     |
| [The Distributed Discrete Gaussian Mechanism for Federated Learning with Secure Aggregation](https://arxiv.org/abs/2102.06387)                   |
| [Training Federated GANs with Theoretical Guarantees: A Universal Aggregation Approach](https://arxiv.org/abs/2102.04655)                        |
## 02.01 - 02.05


| Title                                                                                                                                                             |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Curse or Redemption? How Data Heterogeneity Affects the Robustness of Federated Learning](https://arxiv.org/abs/2102.00655)                                      |
| [Provably Secure Federated Learning against Malicious Clients](https://arxiv.org/abs/2102.01854)                                                                  |
| [A Bayesian Federated Learning Framework with Multivariate Gaussian Product](https://arxiv.org/abs/2102.01936)                                                    |
| [Privacy-Preserving Blockchain-Based Federated Learning for IoT Devices](https://arxiv.org/abs/1906.10893)                                                        |
| [Analysis and Optimal Edge Assignment For Hierarchical Federated Learning on Non-IID Data](https://arxiv.org/abs/2012.05622)                                      |
| [Decentralized Federated Learning Preserves Model and Data Privacy](https://arxiv.org/abs/2102.00880)                                                             |
| [Distributed Conditional Generative Adversarial Networks (GANs) for Data-Driven Millimeter Wave Communications in UAV Networks](https://arxiv.org/abs/2102.01751) |
| [FedAUX: Leveraging Unlabeled Auxiliary Data in Federated Learning](https://arxiv.org/abs/2102.02514)                                                             |
| [Federated Learning at the Network Edge: When Not All Nodes are Created Equal](https://arxiv.org/abs/2101.01995)                                                  |
| [Federated Learning in Smart Cities: A Comprehensive Survey](https://arxiv.org/abs/2102.01375)                                                                    |
| [Federated Learning on Non-IID Data Silos: An Experimental Study](https://arxiv.org/abs/2102.02079)                                                               |
| [Federated Variance-Reduced Stochastic Gradient Descent with Robustness to Byzantine Attacks](https://arxiv.org/abs/1912.12716)                                   |
| [FedProf: Optimizing Federated Learning with Dynamic Data Profiling](https://arxiv.org/abs/2102.01733)                                                            |
| [FEDZIP: A Compression Framework for Communication-Efficient Federated Learning](https://arxiv.org/abs/2102.01593)                                                |
| [R-GAP: Recursive Gradient Attack on Privacy](https://arxiv.org/abs/2010.07733)                                                                                   |
| [SAFELearning: Enable Backdoor Detectability In Federated Learning With Secure Aggregation](https://arxiv.org/abs/2102.02402)                                     |
| [Scaling Federated Learning for Fine-tuning of Large Language Models](https://arxiv.org/abs/2102.00875)                                                           |
| [Variational Federated Multi-Task Learning](https://arxiv.org/abs/1906.06268)                                                                                     |
| [Bias-Variance Reduced Local SGD for Less Heterogeneous Federated Learning](https://arxiv.org/abs/2102.03198)                                                     |
| [DEAL: Decremental Energy-Aware Learning in a Federated System](https://arxiv.org/abs/2102.03051)                                                                 |
| [Double Momentum SGD for Federated Learning](https://arxiv.org/abs/2102.03970)                                                                                    |
| [Federated Depression Detection from Multi-SourceMobile Health Data](https://arxiv.org/abs/2102.09342)                                                            |
| [Federated Learning on the Road: Autonomous Controller Design for Connected and Autonomous Vehicles](https://arxiv.org/abs/2102.03401)                            |
| [Federated mmWave Beam Selection Utilizing LIDAR Data](https://arxiv.org/abs/2102.02802)                                                                          |
| [Federated Reconstruction: Partially Local Federated Learning](https://arxiv.org/abs/2102.03448)                                                                  |
| [Learning Rate Optimization for Federated Learning Exploiting Over-the-air Computation](https://arxiv.org/abs/2102.02946)                                         |
| [Multi-Tier Federated Learning for Vertically Partitioned Data](https://arxiv.org/abs/2102.03620)                                                                 |
| [Semi-Synchronous Federated Learning](https://arxiv.org/abs/2102.02849)                                                                                           |
| [Towards Generalized and Distributed Privacy-Preserving Representation Learning](https://arxiv.org/abs/2010.01792)                                                |
